app/assets/stylesheets/provider/_patternfly_ie11.scss

```diff
@@ -408,3 +411,26 @@ button:-moz-focusring,
     -ms-grid-column: 2;
   }
 }
+
+.pf-c-nav.pf-m-dark .pf-c-divider {
+  background-color: #4f5255; }
+.pf-c-nav.pf-m-dark .pf-c-nav__item.pf-m-current {
+  /* stylelint-disable-next-line */ }
```

Comment:
Just to keep consistency, we are using this format to disable css linter:

`// scss-lint:disable SelectorFormat`

Like [here](https://github.com/3scale/porta/blob/master/app/assets/stylesheets/provider/_commons.scss#L240)